.github/workflows/py-build.yml

```diff
@@ -1,37 +1,59 @@
-name: Python cryoet_data_portal build
+name: Python cryoet_data_portal release
 
 on:
-  pull_request:
-    paths:
-      - "client/python/**"
-      - ".github/workflows/**" # Re-run if a workflow is modified - useful to test workflow changes in PRs
   push:
-    branches: [main]
+    branches:
+      - main
```

Comment:
We shouldn't publish for each push to main. We should have this workflow be runnable on dispatch, and should limit the people who can allow this to be executed.

Reply:
> We shouldn't publish for each push to main.

The release of the API is gates by https://github.com/chanzuckerberg/cryoet-data-portal/blob/8fcc103fceebcfd18c5b94f3ee14baa8239f83d6/.github/workflows/py-build.yml#L27-L27
Which is only true when a release PR is merged into main for the client. So who ever has permission to merge a PR has permission to release the client.

>  limit the people who can allow this to be executed

Who should the release be limited to? As the PR is currently written, you need permission to merge a PR to release the client.

Reply:
> So who ever has permission to merge a PR has permission to release the client.

Currently all the contributors can merge PR. I think that should be fine as that limits unknown external actors from kicking of a deploy.